Sources/Testing/Traits/TimeLimitTrait.swift

```diff
@@ -39,18 +63,123 @@ extension Trait where Self == TimeLimitTrait {
   ///
   /// - Returns: An instance of ``TimeLimitTrait``.
   ///
+  /// Test timeouts do not support high-precision, arbitrarily short durations,
+  /// due to variability in testing environments. The time limit duration must
```

Comment:
```suggestion
  /// due to variability in testing environments. The time limit must
```

A time limit is already, English-wise, a duration, so no need to restate it in prose.